src/inbox.rs

```diff
@@ -301,7 +312,10 @@ impl<A> Chan<A> {
 struct ChanInner<A> {
     capacity: Option<usize>,
     ordered_queue: VecDeque<Box<dyn MessageEnvelope<Actor = A>>>,
-    waiting_senders: VecDeque<Weak<Spinlock<WaitingSender<SentMessage<A>>>>>,
+    waiting_send_to_one:
+        VecDeque<Weak<Spinlock<WaitingSender<Box<dyn MessageEnvelope<Actor = A>>>>>>,
+    waiting_send_to_all:
+        VecDeque<Weak<Spinlock<WaitingSender<Arc<dyn BroadcastEnvelope<Actor = A>>>>>>,
     waiting_receivers_handles: VecDeque<FulfillHandle<A>>,
```

Comment:
It might be interesting (now or in another PR) to introduce 3 here - 1 per queue - which use the queue's type. So, the ordered queue gets a VecDeque, and priority queues get BinaryHeaps. This should preserve ordering behaviour for rendezvous channels.

Reply:
I'd prefer to postpone this, esp. because I'd like to discuss https://github.com/Restioson/xtra/issues/94 again.

Reply:
Of course :)